state/filenames.go

```diff
@@ -15,5 +15,7 @@
 package state
 
 const (
-	ClusterFile = "cluster.yaml"
+	ClusterFile     = "cluster.yaml"
+	ClusterJsonFile = "api.conf"
```

Comment:
Can we stick with the `cluster.$encoding` format here? So we would use `cluster.json`?
The reason for this is for when we build snapshots (in the future) we are going to want the files named to represent the cluster, and the apps on top.

Also if we are introducing new filenames, can we patch the old constant to use `ClusterYamlFile` to keep the convention throughout the package?